src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int TODO_DONE_LENGTH = 5;
```

Comment:
Perhaps you could consider separating "TODO_DONE_LENGTH" to two separate constants of "TODO_LENGTH" and "DONE_LENGTH"? This may increase code readability. I noticed the similar naming style in line 8 of your code.

Reply:
I agree with Hazel on this, the same could be done for BY_AT_LENGTH as well.

Reply:
thank you both, will update my code